### PR TITLE
[Prim] fix eval branch of composite rule of batch_norm

### PR DESCRIPTION
**Pull request by cyber-pioneer**
### PR types
Bug fixes

### PR changes
Others

### Describe
Pcard-66969
Fix bug: eval branch of composite rule of batch_norm is different of class BatchNorm.

### CODE REVIEW

**Review by JiabinYang — APPROVED** [review]
LGTM
